rs/registry/canister/chunks/src/lib.rs

```diff
@@ -0,0 +1,58 @@
+use ic_crypto_sha2::Sha256;
+use ic_stable_structures::{Memory, StableBTreeMap};
+
+const CHUNK_SIZE_BYTES: usize = 1_800_000;
```

Comment:
Should this be something that can be determined by the client?

Reply:
I thought about it originally, but

1. I doubt it would be really useful in practice. If we assume there will be many users (seems overly optimistic), I think they are going to want something close to (but less than) 2e6.
2. If we later find that it's useful, we can add it.
3. This is simpler.